cardano-api/src/Cardano/Api/Certificate.hs

```diff
@@ -108,7 +108,7 @@ instance FromCBOR Certificate where
     fromCBOR = fromShelleyCertificate <$> fromCBOR
 
 instance HasTextEnvelope Certificate where
-    textEnvelopeType _ = "CertificateShelley"
```

Comment:
`"CertificateShelley"' and the other `textEnvelopeType` names are all magic IDs and they are assumed to be unique throughout the code.
It would be better to have all those unique ID in one place.

Reply:
What do you have in mind?